.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+- '5.7'
+cache:
+  directories:
+  - node_modules
+after_success: npm run coverage
+notifications:
+  slack:
+    secure: dWSrlBtTp58mjFtp7BGmNITbutowrV4TT9W9k35/IHLkxjTF3rG+21KsGgEOAZBSnz63g1OOxwopJ0lClWtWR1E5HVwlCtbES7ZnCU1BuqyZ3IbQvT+2wzcJBdcagKBb03+YVQOJng1uh3XhbhF664DS7RLZOQ5885r7ks/Ic+j9kBqcesNJC0XgGmP3iZOqD29z9aPUs58kvLeupmqWYm1E2Zx8JXjP+vUoe7qj36QzY39u+GRPHqdEUSnIQS0kzoSTNDdWWOGl+zu6BYY/6Xxy6LorEuRfx9slxpyoSdKgtQXblHDTGwBLQTtOD4mGfkwrapKfDRWV9jT+wr6q94bqKkNQ4NR3L5wouvI6j+bslxD8w/7XZOsvEUv9KSrGsRbvO1Dedi9gFuU/8q5VuBE+Q3mQ9lSaEEBioker7W3Byq5JSDHruisuH6o3B2WPA7lkPuAfX0LnUt+5by2yodwk4t+aXNx1VD9ZK592A5h8XV7IRRx5V5w2H4+kzzO3ZEAjUv6VU8YyTKO3k4ngD5DmB28vc1hVVd/Pt7gSbQBQoUBoxWkZed3TNLHqWlsiuXxYy3a8aJd5UFefp4diUfljbf6zOgvs8P0JpIH1wzTm0QDlAf6bbl1ERa2C3mKgSP6MyC425RcUqvOzc4n6TCLl6kYizw+RxtjB447SIOM=
```

Comment:
Where does this come from? Did it come from using the `travis` cli or was added in there from the generator?